eventuals/conditional.h

```diff
@@ -105,11 +106,19 @@ struct _Conditional {
         "\"then\" and \"else\" branch of 'Conditional' *DO NOT* return "
         "an eventual value of the same type");
 
-    using ThenAdapted_ = decltype(std::declval<ThenE_>().template k<void>(
-        std::declval<_Then::Adaptor<K_>>()));
+    using ThenAdapted_ =
+        decltype(std::declval<
+                     ThenE_>()
+                     .template k<void, Errors_>(
+                         std::declval<
+                             _Then::Adaptor<K_>>()));
 
-    using ElseAdapted_ = decltype(std::declval<ElseE_>().template k<void>(
-        std::declval<_Then::Adaptor<K_>>()));
+    using ElseAdapted_ =
+        decltype(std::declval<
+                     ElseE_>()
+                     .template k<void, Errors_>(
+                         std::declval<
+                             _Then::Adaptor<K_>>()));
```

Comment:
For less whitespace can you not wrap the `std::declval<ThenE_>()` and `std::declval<ElseE_>()`?